modules/rosa-sts-understanding-ocm-role.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// rosa_architecture/rosa-sts-about-iam-resources.adoc
+//
+:_content-type: CONCEPT
+[id="rosa-sts-understanding-ocm-role_{context}"]
+= Understanding the {cluster-manager} role
+
+Creating ROSA clusters in {cluster-manager-url} require an `ocm-role` IAM role. The basic `ocm-role` IAM role permissions let you to perform cluster maintenance within {cluster-manager}. To automatically create the operator roles and OpenID Connect (OIDC) provider, you must add the `--admin` option to the `rosa create` command. This command creates an `ocm-role` resource with additional permissions needed for administrative tasks.
+
+[NOTE]
+====
+This elevated IAM role allows {cluster-manager} to automatically create the cluster-specific Operator roles and OIDC provider during cluster creation. For more information about this automatic role and policy creation, see the "Understanding the auto and manual deployment modes" link in Additional resources.
+====
+
+[id="rosa-sts-understanding-user-role_{context}"]
+== Understanding the user role
+
+In addition to an `ocm-role` IAM role, you must create a user role so that {product-title} can verify your AWS identity. This role has no permissions, and it is only used to create a trust relationship between the installer account and your `ocm-role` resources.
```

Comment:
Should "the installer account" be "the installation program" account based on the style guide. I am not sure that changes the meaning here or not.